Lib/ufo2ft/outlineCompiler.py

```diff
@@ -118,6 +121,7 @@ def __init__(
         self._glyphBoundingBoxes = None
         self._fontBoundingBox = None
         self._compiledGlyphs = None
+        self.compileTrueTypeHinting = compileTrueTypeHinting
```

Comment:
I would expect `compileTrueTypeHinting` to only be available for the TrueType outline compiler, not for the base compiler class.